docs: Misc updates for HR (#14181)
unoplatform/uno: doc/articles/get-started-wizard.md

#### The XAML editor shows `The type 'page' does not support direct content` message

This issue has been fixed in Visual Studio 17.8 Preview 5 and later.

> If you're using an ealier version, XAML Intellisense [is not working properly](https://developercommunity.visualstudio.com/content/problem/587980/xaml-intellisense-does-not-use-contentpropertyattr.html) in Visual Studio when the active project is not the UWP one.

Reviewer: ```suggestion
If you're using an earlier version, XAML Intellisense [is not working properly](https://developercommunity.visualstudio.com/content/problem/587980/xaml-intellisense-does-not-use-contentpropertyattr.html) in Visual Studio when the active project is not the UWP one.
```

To work around this issue, close all XAML editors, open a C# file and select the '[MyApp].Windows' in the top-left drop-down list of the text editor sector. Once selected, re-open the XAML file.

#### `InitializeComponent` or `x:Name` variable is not available in code-behind

This issue has been fixed in Visual Studio 17.8 Preview 5 and later.

> If you're using an ealier version, Visual Studio [does not refresh the intellisense cache](https://developercommunity.visualstudio.com/content/problem/588021/the-compile-itemgroup-intellisense-cache-is-not-re.html) properly, causing variables to be incorrectly defined.

Reviewer: ```suggestion
If you're using an earlier version, Visual Studio [does not refresh the intellisense cache](https://developercommunity.visualstudio.com/content/problem/588021/the-compile-itemgroup-intellisense-cache-is-not-re.html) properly, causing variables to be incorrectly defined.
```

To fix this issue, build your project once, close the solution and reopen it.

> It is also important to note that Uno Paltform uses a multi-project structure, for which each project has to be build individually for errors to disappear from the **Error List** window (notice the **Project** column values).

Reviewer: ```suggestion
It is also important to note that Uno Platform uses a multi-project structure, for which each project has to be built individually for errors to disappear from the **Error List** window (notice the **Project** column values).
```

On Linux, you'll need to follow the [Uno Platform](get-started-with-linux.md#setting-up-for-linux) setup instructions.
On macOS, you'll need to follow the [Uno Platform](get-started-vsmac.md) setup instructions.

#### Hot Reload troubleshooting

> The Hot reload provides a Visual Studio for Windows output window name "Uno Platform" with diagnostics messages. You can find additional information there in case XAML Hot Reload does not work properly.

Reviewer: ```suggestion
The Hot Reload provides a Visual Studio for Windows output window named "Uno Platform" with diagnostics messages. You can find additional information there in case XAML Hot Reload does not work properly.
```

Some common troubleshooting steps:
- Make sure to rebuild your application if the XAML changes are not applied

> - Ensure that the Uno.WinUI.DevServer package has the same version as the Uno.WinUI package (Similar step is valid for Uno.UI packages)

Reviewer: ```suggestion
- Ensure that the Uno.WinUI.DevServer package has the same version as the Uno.WinUI package (A similar step is valid for Uno.UI packages)
```
